front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -23,6 +24,20 @@ function formatChCode(chCode: string) {
   return chCode === '' ? 'BV' : chCode;
 }
 
+const extractChCodes = (searchResults: SearchResultItemOperationalPoint[], selectedCI: CIOption) =>
+  searchResults
+    .filter((pr) => pr.name === selectedCI.name)
```

Comment:
```suggestion
    .filter((op) => op.name === selectedCI.name)
```
I think we should use "op" instead of "pr" that is a French acronym

Reply:
Good catch !